homework-submission/week1-react/src/component/todo.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import changeDateFormat from '../util/change-format-date.js';
+
+class Todo extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
```

Comment:
I see you have forgotten this in here while you are not using it. 😅

Reply:
Yepp :)